### DISCUSSION TITLE
Hint action populate component

### DISCUSSION THREAD

**Discussion by clinvest** [filamentphp/filament#11484] [Help]
### Package

Panel builder

### Package Version

v3.0.0

### How can we help you?

Hi all,

I'm populating a select field from a hint action, it's working but it displays the 'id' of the new model instead of the label. I tried to populate options also in addition, but I don't find the solution. Any idea?

<img width="996" alt="Capture d’écran 2024-02-19 à 17 28 41" src="https://github.com/filamentphp/filament/assets/144620798/444c1142-32b0-4154-9407-7ce793097daf">

And my code:

```
ExtendedSelect::make('invoices_recipients')
    ->populateEntityInvoicesRecipients($arguments)
    ->searchableContact()
    ->disabled($disabled)
    ->visible($entityableType === Organizer::class)
    ->multiple()
    ->hintAction(ContactCreateHintAction::make())
    ->live()
    ->label(__('app.invoicing_contacts'))->required(),
```

From my hint action:

```
$this->action(function (Form $form): void {

            $this->process(function (array $data, $state, $component = null) {

                $contact = Contact::make($data);
                $contact->save();

                $state[] = $contact->id;
                $component->state($state);

                /*
                $options = $component->getOptions();
                array_push($options, [$contact->id => $contact->email]);
                $component->options($options);
                */

            });

            $this->success();

        });
```

**Comment by leandrocfe** [comment]
Why not [createOptionForm](https://filamentphp.com/docs/3.x/forms/fields/select#creating-a-new-option-in-a-modal)?
